test/components/FeatureFlagRenderer.test.tsx

```diff
@@ -47,16 +55,16 @@ describe("components/FeatureFlagRenderer", () => {
   );
 
   beforeEach(() => {
-    utils.ldClientWrapper = jest.fn();
-    utils.ldClientWrapper.mockImplementation(() => ({
+    (utils.ldClientWrapper as jest.Mock) = jest.fn();
```

Comment:
Did mocked that we looked at the other day not an option? Not an issue just out of interest

Reply:
From what I tried, I still had to use as `jest.Mock` when mocking the function and only then can use `mocked` to mock implementation or other things, so I taught it's better to keep one to not add confusion.

It's possible that I didn't do it right, but honestly once I saw the tests passing that day I was happy to stay with `jest.Mock`. Might be worth giving it a try in our projects :D